colossalai/fx/tracer/_tracer_utils.py

```diff
@@ -29,3 +31,18 @@ def _convert(val):
     new_args = [_convert(val) for val in args]
     new_kwargs = {k: _convert(v) for k, v in kwargs.items()}
     return new_args, new_kwargs
+
+
+def compute_meta_data_for_functions_proxy(target, args, kwargs):
+    args_metas, kwargs_metas = extract_meta(*args, **kwargs)
+
+    # fetch patched function
+    if meta_patched_function.has(target):
+        meta_target = meta_patched_function.get(target)
+    else:
+        meta_target = target
```

Comment:
Some functions are indexed by name, e.g. torch builtin add is registered with the name 'add' as it can cover `torch.add`, `torch.Tensor.add` and `object.__add__`, so add this to the if statement.

```python
if meta_patched_function.has(target):
        meta_target = meta_patched_function.get(target)
elif meta_patched_function.has(target.__name__):
        meta_target = meta_patched_function.get(target.__name__)
else:
        meta_target = target
```